docs/victoriametrics-cloud/_index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: VictoriaMetrics Cloud Overview
+title: VictoriaMetrics Cloud
 weight: 40
 disableToc: true
```

Comment:
most likely you what to remove this line

Reply:
Not really. Sorry for the confusion. It was a mistake from previous PR. The "Overview" in the title here is making the menu in the left to look wrong, as it looks the . The "Overview" file makes the title fit when needed.

Reply:
i meant disableToc line

Reply:
Thanks! I see no difference in my local deployment, but maybe it changes in the new version. (For the record, is was there before).

Adding the commit on the fly!

```suggestion
```